src/components/pages/AdminAddCourse/AdminAddCoursesForm.js

```diff
@@ -112,10 +116,19 @@ function AdminAddCoursesForm(props) {
   };
 
   const handleCheck = e => {
-    daysOfWeek.push(e.target.value);
-    const dow = daysOfWeek.map(day =>
-      !daysOfWeek.includes(day) ? Array().push(day) : null
-    );
+    if (e.target.checked) {
+      setFormValues({
+        ...formValues,
+        days_of_week: [...formValues.days_of_week, e.target.value],
+      });
+    } else {
+      setFormValues({
+        ...formValues,
+        days_of_week: formValues.days_of_week.filter(
+          value => value !== e.target.value
+        ),
+      });
+    }
```

Comment:
Great work adjusting the implementation of this function and the funky `.map()` we had before!